docs/pages/reference/backends.mdx

```diff
@@ -620,6 +620,282 @@ Config option priority is applied in the following order:
+            ],
+            "Effect": "Allow",
+            "Resource": "arn:aws:kms:eu-central-1:1234567890:key/kms_id",
+            "Sid": "AllowAthenaKMSUsage"
+        }
+    ]
+}
+```
+
+### Migration from Dynamo to Athena audit logs backend
+
+<Admonition
+  type="tip"
+  title="Tip"
+>
+  Migration is only needed if Dynamo audit logs was used and you want to keep
+  old data.
+</Admonition>
+
+Migration consist of following steps:
+
+1. Set up Athena infrastructure
+1. Dual write to both Dynamo and Athena, query from Dynamo
+1. Migrate old data from Dynamo to Athena
+1. Dual write to both Dynamo and Athena, query from Athena
+1. Disable writing to Dynamo
+
+In teleport storage config, `audit_events_uri` accepts multiple URLs used as
+audit loggers. If more than 1 is used, then events are written to each audit
+system, and queries are executed from first one.
+
+<Admonition
+  type="tip"
+  title="Tip"
+>
+  If anything goes wrong during migration steps 1-4, rollback to Dynamo solution
+  is as trivial as making sure it's on first place inside `audit_events_uri`
+  and removing Athena URL from `audit_events_uri`.
+</Admonition>
+
+#### Dual write to both Dynamo and Athena, query from Dynamo
+
+Second step of migration requires setting following configuration:
+
+```yaml
+teleport:
+  storage:
+    audit_events_uri: 
+    - 'dynamodb://events_table_name'
+    - 'athena://db.table?otherQueryParams'
+```
+
+When auth server is restarted, you should verify that parquet files are stored
+on S3 bucket specified using `locationS3` parameter.
+
+#### Migrate old data from Dynamo to Athena
+
+This step requires using client machine to export data from Dynamo and publish
+it to Athena logger. We recommend using for example EC2 instance with disk size
+at least 2x bigger than table size in dynamo.
+
+Instructions how to use migration tool can be found
+[here](https://github.com/gravitational/teleport/blob/master/examples/dynamoathenamigration/README.md).
+
+You should set `exportTime` to time when dual writing was started.
+
+It is recommended to first run migration with `-dry-run` flag, because it
+allows to run validation of exported data. If no errors are reported, proceed
+to real migration without `-dry-run` flag.
+
+#### Dual write to both Dynamo and Athena, query from Athena
+
+Change order of `audit_events_uri` in teleport configuration:
+
+```yaml
+teleport:
+  storage:
+    audit_events_uri:
+    - 'athena://db.table?otherQueryParams'
+    - 'dynamodb://events_table_name'
+```
+
+When auth server is restarted, you should verify that events are visible on
+Audit Logs page.
+
+#### Disable writing to Dynamo
+
+<Notice type="danger">
+
+Disabling writing to Dynamo means that you won't be able to rollback to Dynamo
+without loosing data. Dual writing to both Athena and Dynamo does not have big
+performance impact and it's recommended to keep dual writing for some time, even
+if queries are done already from Athena.
+
+</Notice>
```

Comment:
```suggestion
Disabling writing to DynamoDB means that you won't be able to roll back to DynamoDB
without losing data. Dual writing to both Athena and DynamoDB does not have a significant
performance impact, and it's recommended to keep dual writing for some time, even
if your system already executes queries from Athena.
```

- Removing the `Notice` tags since the `Notice` encompasses most of this section and becomes distracting.
- Grammar suggestions.